src/components/Paragraph/index.js

```diff
@@ -3,14 +3,64 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import { useTheme } from 'themes';
+import { bodySmall, bodyMedium, bodyLarge } from 'themes/mixins';
 
-export const Paragraph = ({ children, ...otherProps }) => {
+export const Paragraph = ({
+  children,
+  large,
+  small,
+  inverse,
+  dark,
+  light,
+  ...otherProps
+}) => {
   const theme = useTheme();
 
   return (
     <p
       css={css`
-        font-family: ${theme.typography.bodyFont};
+        ${css(bodyMedium(theme))};
+        color: ${theme.colors.text.default};
+        margin: 0 0 4rem;
+
+        &:last-of-type {
+          margin-bottom: 0;
+        }
+
+        ${large &&
+          css`
+            ${css(bodyLarge(theme))};
+          `}
+
+        ${small &&
+          css`
+            ${css(bodySmall(theme))};
+          `}
+
+          ${light &&
```

Comment:
These are indented a bit farther than I'd expect, eh? I guess I need to improve the style linting/prettier settings.

Reply:
Yeah, the indentation on these is super weird and hard to follow. Happy to see a fix!